packages/cactus-plugin-ledger-connector-corda/src/main/json/openapi.json

```diff
@@ -866,7 +866,341 @@
       "PrometheusExporterMetricsResponse": {
         "type": "string",
         "nullable": false
+      },
+      "listCpiV1Response":{
```

Comment:
@adrianbatuto Please have the type names capitalized (applies to all of the types in the Typescript code and also in all openapi.json files across the board)